src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Importing/Exporting Workflows into the Catalog
+
+In the Workflow Catalog, it is possible to select Workflows from the current bucket and send them to another bucket, which can belong to the same Catalog or to another Scheduler's Catalog.
+
+==== Send Workflows directly
+
+It is convenient to send the Workflows to share directly, if they are sent to a Scheduler with a public API, or to another bucket of the local Scheduler.
+
+Select the bucket then the Workflows. Choose `Send to another scheduler` and click `OK`.
```

Comment:
You don't describe the difference in the procedure between sending to another scheduler or to the same scheduler on another bucket. Is there not an url you must enter ? In that case, you could maybe provide an example of the two use cases.